src/components/Input.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface InputProps extends React.ComponentProps<'input'> {
```

Comment:
Cool, I didn't know about this way of typing the input field. I used to use something like:
`interface InputProps extends React.InputHTMLAttributes<HTMLInputElement>`